File: Daily Test/C++/Count Cells - Coordinates Values 8.7.2025.cpp
#include <iostream>
using namespace std;

int main() {
    int r, c, m[50][50], cnt = 0;
    cin >> r >> c;
    for (int i = 0; i < r; ++i)
        for (int j = 0; j < c; ++j)
            cin >> m[i][j];

    for (int i = 0; i < r; ++i)
        for (int j = 0; j < c; ++j)
            if (m[i][j] == (i+1)*10 + (j+1)) cnt++;

    cout << cnt << endl;
    return 0;
}

File: Daily Test/Java/Count Cells - Coordinates Values 8.7.2025.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int r = sc.nextInt(), c = sc.nextInt();
        int[][] m = new int[r][c];
        int cnt = 0;

        for (int i = 0; i < r; ++i)
            for (int j = 0; j < c; ++j)
                m[i][j] = sc.nextInt();

        for (int i = 0; i < r; ++i)
            for (int j = 0; j < c; ++j)
                if (m[i][j] == Integer.parseInt("" + (i+1) + (j+1)))
                    cnt++;

        System.out.println(cnt);
    }
}